domain/src/main/java/org/oppia/android/domain/devoptions/ModifyLessonProgressController.kt

```diff
@@ -101,6 +103,74 @@ class ModifyLessonProgressController @Inject constructor(
     return true
   }
 
+  /**
+   * Marks multiple topics completed given a profile ID and list of topic IDs.
+   *
+   * @param profileId the ID corresponding to the profile for which progress needs modified.
```

Comment:
small nit: 
```suggestion
   * @param profileId: the ID corresponding to the profile for which progress needs modified.
```
ditto elsewhere.

Reply:
Done.